tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "include": ["**.js"],
+  "compilerOptions": {
+    "target": "es2020",
+    "lib": ["es2020"],
```

Comment:
should we start higher than 2020?

Reply:
I think `es2020` is fine. https://github.com/tsconfig/bases/blob/main/bases/node14.json

Reply:
sure!